src/lib/components/Tag.svelte

```diff
@@ -15,22 +15,22 @@
 <style lang="scss">
   .tag {
     // "info" intent is the default
-    background-color: var(--elements-divider);
-    color: var(--text-description);
+    background-color: var(--tag-background);
+    color: var(--tag-text);
 
     &.success {
       background-color: var(--positive-emphasis);
-      color: var(--text-light);
+      color: var(--tag-text-light);
     }
 
     &.warning {
       background-color: var(--warning-emphasis);
-      color: var(--text-text);
+      color: var(--tag-text-dark);
```

Comment:
Where do we use the `warning` and `error` tags and what do they look like?

Reply:
I don’t think those styles are used in the NNS dapp, so they’re here for backward compatibility.

<img width="347" alt="image" src="https://github.com/user-attachments/assets/1bca68e8-ebf2-4bfa-aab0-d28f54c6de81">